### PR TITLE
Interquartile range calculator added with steps

### LINKED ISSUE #4415: Interquartile Range Calculator
I'd like to add this calculator.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
## Related Issue

- Info about Issue or bug
Fixes #4415 
fix #[issue number that will be closed through this PR]

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
I have added the interquartile range calculator.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by sairish2001 — CHANGES_REQUESTED** [review]
Actually you are computing and adding data into div inside the function itself, we are planning to release this as a library also, so kindly check `function solveFraction (num1,num2);` i.e how we did here, i.e. only returning a object with result and steps, and calling this function in the file `calculators/solveFraction.html`

**Comment by Khushi-agarwal** [comment]
@sairish2001 done the changes.

File: library/logicjavascript.js [hunk 1]
```diff
@@ -29156,4 +29156,26 @@ function solveFraction(num1, num2) {
...
+    ans+="Enter all values";}
+    else
+    {
+        ans+="Step3: Calculate the inter quartile range";
+        
+        iq=q3-q1;
+        ans+="The Interquartile range of"+$q3$-$q1$;
+        ans+=iq;   
+    }
+    return{'result':iq,'steps:':ans};
```
Comment (sairish2001) [inline library/logicjavascript.js:29180]: you wrote 'steps:' : inside the double quotes, use 'steps' not 'steps:'

**Review by sairish2001 — CHANGES_REQUESTED** [review]

File: library/logicjavascript.js (hunk 1 above)
Comment (Khushi-agarwal) [reply library/logicjavascript.js:29180]: > you wrote 'steps:' : inside the double quotes, use 'steps' not 'steps:'

done.

**Review by sairish2001 — APPROVED** [review]
Great work :)

**Pull request merged**